packages/components/src/internal/components/search/SearchPanel.tsx

```diff
@@ -0,0 +1,216 @@
+                                <i className="fa fa-search search-icon" />
+                            </span>
+                            <input
+                                className="form-control search-input"
+                                onChange={onSearchChange}
+                                placeholder="Search"
+                                size={34}
+                                type="text"
+                                value={searchQuery}
+                            />
+                        </span>
+                    </form>
+                    <Button type="submit" className="margin-left success submit-button" onClick={onSearchClick}>
+                        Search
+                    </Button>
+                    {hasPages && (
+                        <div className="page-buttons">
+                            <PaginationButtons
+                                total={totalHits}
+                                currentPage={currentPage}
+                                perPage={pageSize}
+                                previousPage={pageBack}
+                                nextPage={pageForward}
+                            />
+                        </div>
+                    )}
+                </div>
+                {searchTerm && (
+                    <SearchResultsPanel
+                        model={searchResultsModel}
+                        hidePanelFrame={true}
+                        emptyResultDisplay={emptyTextMessage}
+                        offset={offset}
+                    />
+                )}
+            </Section>
+        </Page>
+    );
+});
+
+interface SearchPanelProps {
+    getCardDataFn: GetCardDataFn;
+    offset?: number; // Result number to start from
+    pageSize?: number; // number of results to return/display
+    search: (form: any) => any;
+    searchTerm: string;
+}
+
+export const SearchPanel: FC<SearchPanelProps> = memo(props => {
+    const { searchTerm, getCardDataFn, search, pageSize = SEARCH_PAGE_DEFAULT_SIZE, offset = 0 } = props;
+    const [searchQuery, setSearchQuery] = useState<string>(searchTerm);
+    const [model, setModel] = useState<SearchResultsModel>(SearchResultsModel.create({ isLoading: true }));
+
+    const loadSearchResults = useCallback(async () => {
+        if (searchQuery) {
+            setModel(SearchResultsModel.create({ isLoading: true }));
+            try {
+                const entities = await searchUsingIndex(
+                    {
+                        experimentalCustomJson: true, // will return extra info about entity types and material results
+                        normalizeUrls: true, // this flag will remove the containerID from the returned URL
+                        q: searchQuery,
+                        scope: SearchScope.Folder, // TODO should this be a parameter/prop?   // only using folder for this application
```

Comment:
@labkey-nicka Can you comment on the intended behavior for search in the presence of projects? Are we meant to search locally or also in the parent and /Shared containers?

Reply:
@labkey-nicka Never mind. I see the `getSearchScopeFromContainerFilter` mehtod.